cmake/Reprojucer.cmake

```diff
@@ -53,6 +53,8 @@ function(jucer_project_begin)
     endif()
     get_filename_component(project_dir "${_PROJECT_FILE}" DIRECTORY)
     set(JUCER_PROJECT_DIR "${project_dir}" PARENT_SCOPE)
+  else()
+    set(JUCER_PROJECT_DIR "${CMAKE_CURRENT_SOURCE_DIR}" PARENT_SCOPE)
```

Comment:
@scotchi since you don't have a `.jucer` file, you indeed don't have a `PROJECT_FILE` to give to `jucer_project_begin()`. However, there is nothing preventing your from defining `JUCER_PROJECT_DIR` manually in your CMakeLists.txt:
```cmake
set(JUCER_PROJECT_DIR "<project_dir>")
```
where `<project_dir>` is `${CMAKE_CURRENT_LIST_DIR}`, `${CMAKE_CURRENT_SOURCE_DIR}`, `${CMAKE_SOURCE_DIR}`, or something else, depending on the structure of your JUCE project. I added that option to the error message from `_FRUT_abs_path_based_on_jucer_project_dir()` in #350.